File: homework-nonnative/lib/types/product.dart
import 'package:uuid/uuid.dart';

class Product {
  final String id = const Uuid().v4();
  String title;
  String category;
  String image;
  double price;
  int popularity;
  int quantityLeft;

  Product({
    required this.title,
    required this.category,
    required this.image,
    required this.price,
    required this.popularity,
    required this.quantityLeft
  });
}

File: homework-memory/homework/Types/Product.swift
//
//  Product.swift
//  homework
//
//  Created by Nita Horia on 16.11.2022.
//

import Foundation

struct Product: Identifiable {
    var id: String = UUID().uuidString
    var category: String
    var image: String
    var price: Float
    var title: String
    var popularity: Int
    var quantityLeft: Int
}
